wandb/apis/public/api.py

```diff
@@ -1522,6 +1524,95 @@ def registries(
         )
         return Registries(self.client, organization, filter)
 
+    def registry(self, name: str, organization: Optional[str] = None) -> Registry:
```

Comment:
Going to leave name as without `wandb-registry-` first. if people complain we can go about fixing it. Mainly because i don't want to encourage bad behavior from the get go

Reply:
I think this makes sense as is because we're adding the registry API as an abstraction over a project. 🤞